plugins/org.yakindu.sct.doc.user/src/user-guide/sctunit.textile

```diff
@@ -299,9 +299,9 @@ bc.
 
 ==</div>==
 
-p. The _header_ of an operation consists of an optional annotation, typically <code>&#x40;Test</code>, followed by the keyword @operation@, followed by the name of the operation, followed by a parenthesized list of parameters, optionally followed by a colon and a return type.
+p. The _header_ of an operation consists of an optional annotation, typically <code>&#x40;Test</code>, followed by the keyword @operation@, followed by the name of the operation, followed by a parenthesized list of parameters, optionally followed by a colon and a return type. Specifying no return type is equivalent to explicitly specifying @void@ as return type.
```

Comment:
I think this is not the case, when no type is specificed it will be inferred by its return statements, like

operation doit() {
return 5
}

var x : integer = doIt() is valid

Reply:
Thanks, fixed.